.github/workflows/linux.yml

```diff
@@ -0,0 +1,36 @@
+name: linux
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build_basic:
+    name: Basic
+    runs-on: ubuntu-latest
+    steps:
+    - name: Install Deps
+      run: |
+           sudo apt-get update
+           sudo apt-get install binutils \
+                                gcc \
+                                g++ \
+                                gfortran \
+                                python \
+                                perl \
+                                git \
+                                git-lfs \
+                                curl \
+                                wget \
+                                tar \
+                                unzip \
+                                build-essential
+    - uses: actions/checkout@v2
+    - name: Run Uberenv
+      run: |
+          cd  .ci/test-project
+          python ../../uberenv.py  --project-json=uberenv_config.json
```

Comment:
Does this fail if uberenv fails? or does it need some logic to fail the CI?

Reply:
the ci fails uberenv fails (or if the cat command fails)